### PR TITLE
Add security e-mail alias

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

**Review by bpholt — APPROVED** [review]

**Comment by bpholt** [comment]
Sorry if you were waiting for my review on this! LGTM.

**Review by ChristopherDavenport — APPROVED** [review]

**Comment by rossabaker** [comment]
Wasn't waiting.  Forgot it was here. 😆

**Pull request merged**
